authentication/email.py

```diff
@@ -0,0 +1,17 @@
+import smtplib
+from authentication import *
+
+def send_email(self, email_receiver):
+    email_sender = 'hammadkhanniazi420@yahoo.com'  # enter your email
+    email_sender_password = 'rdhgdzlwxnkwyqrv'  # enter your password (app-password)
+    subject = 'Email Verification'
```

Comment:
create a file consts, where you can define these constants and import from there.

Reply:
Added